File: 프로그래머스/0/120882. 등수 매기기/등수 매기기.py
def solution(score):
    answer = []
    n_score = []
    for i in score:                          # 평균 모으기
        n_score.append(sum(i)/len(i))       # sum(i)//len(i)라고 했을때 정수부분만 나와서 평균값을 엄밀히 나눌 수 없다.
    
    for i in n_score:                        # 원소는 원래 정렬에서 꺼내고 인덱스는 정렬된 곳에서 꺼내기
        answer.append(sorted(n_score)[::-1].index(i) + 1)
    return answer

File: 프로그래머스/0/120882. 등수 매기기/등수 매기기.java
import java.util.Comparator;
import java.util.HashMap;
import java.util.HashSet;
import java.util.List;
import java.util.ArrayList;
class Solution {
    public int[] solution(int[][] score) {

    int scoreln = score.length;

    List <Integer> box = new ArrayList<>();
    int[] answer = new int[scoreln];

    for (int i =0; i < scoreln; i++){
        box.add(score[i][0] + score[i][1]);
    }

    box.sort(Comparator.reverseOrder());

    for (int i =0; i < scoreln; i++) {
        answer[i] = box.indexOf(score[i][0] + score[i][1]) + 1;
    }
    
    return answer;
}
}